### DISCUSSION TITLE
Flappy Bird Flawed...

### DISCUSSION THREAD

**Discussion by 14Codes** [community/community#75685] [Programming Help]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/147854889/670b7573-a707-4540-bdf2-63f857e4da16)
Not even sure how this happened. I am trying to recreate flappy bird and this is what i got...

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @14Codes! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. I'll keep it closed for now but please feel free to reopen this discussion if you do still need help here. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
